source/specs/fees.rst

```diff
@@ -4,7 +4,7 @@ Transaction Fees
 The Aergo protocol includes transaction fees that need to be paid
 according to the configuration of the network.
 
-Versions prior to 2.0 (`see 1.3 docs <https://docs.aergo.io/en/1.3/specs/fees.html>`_) 
+Versions prior to 2.0 (`see 1.3 docs <https://aergo.readthedocs.io/en/1.3/specs/fees.html>`_)
```

Comment:
it should be `docs.aergo.io`
if the link is not working, then the correct must be found

Reply:
It appears that only links to `/latest/` are working
Links to direct versions are not working, for some reason
We will need to check that